src/components/AudioPlayer/PlayerButtons/Forward/ForwardButton.component.js

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import propTypes from 'prop-types';
+
+import fastForwardIcon from '../../icons/fast-forward.svg';
+
+const ForwardButtonComponent = ({ isDisabled, playNextSong }) => (
+  <button type="button" className={`player-button ${isDisabled ? 'disable-buttons' : ''}`} onClick={playNextSong} disabled={isDisabled}>
```

Comment:
this is a little bit confusing, maybe because of looking this at Github...

```javascript
let buttonStatusClass = isDisabled ? 'disable-buttons' : '';
```

Reply:
I personally dislike ternaries on class names because it is always hard to read. In the end, it is just a simple string builder... could we have one? :P